pom.xml

```diff
@@ -96,7 +96,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-lambda-java-log4j</artifactId>
```

Comment:
Discussed offline `aws-lambda-java-log4j` doesn't have a version 1.5.1, that version is in this package: `aws-lambda-java-log4j2`

Reply:
Tested the build. It runs clean with `aws-lambda-java-log4j2` version 1.5.1

Reply:
yayyyy